data/json/npcs/valhalla_cult/TALK_valhallist_gode.json

```
@@ -0,0 +1,84 @@
+    "responses": [
+      { "text": "What are you a prophet of?", "topic": "TALK_VALHALLIST_GODE_Religion" },
+      { "text": "What is that purpose?", "topic": "TALK_VALHALLIST_GODE_PURPOSE" },
+      { "text": "I have to go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_VALHALLIST_GODE_PURPOSE",
+    "dynamic_line": "That purpose is to serve the true lords of this world, and enter Valhalla on some coming day.  There I will not worry about the wandering dead, but see the might of battle and feast with those worthy.  Odin the Allfather will smile on me that day, and everyone I brought with me.",
+    "responses": [
+      { "text": "Let's talk about something else.", "topic": "TALK_VALHALLIST_GODE_2" },
+      { "text": "Good luck with that.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_VALHALLIST_GODE_Job",
+    "dynamic_line": "This is our holy henge devoted to the gods.  Here we stay between pillaging the ruins of the old world, perform our rites for divine favor, and feast in merriment.  We all help each other in these trying times, united in our divine strength and sharp blades.  I help guide our warriors in battle and feast.",
+    "responses": [
+      { "text": "What are you guiding them toward?", "topic": "TALK_VALHALLIST_GODE_Religion" },
+      { "text": "How could I get in on this?", "topic": "TALK_VALHALLIST_GODE_Join" },
+      { "text": "Enjoy yourself then, I have to go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_VALHALLIST_GODE_Join",
+    "dynamic_line": "Our leader should be around here somewhere, likely in their hall.  While I can teach you about our way, they're the ones who could let you join us.  Gudene vil det!",
+    "responses": [ { "text": "Thanks.", "topic": "TALK_DONE" } ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_VALHALLIST_GODE_Religion",
+    "dynamic_line": "It seems likely you wouldn't know.  Before there was anything in the world, there was fires of Muspelhiem and the frigid ice of Niflheim.  When these two came together, the ice melted and became the first of the giants, Ymir.  As he slept, more giants sprung from his sweat and saliva.  From the melt came the cow Audhumla, who licked away the ice until she uncovered Buri, the first of the gods.  His children were Odin, Vili, and Ve; who slew Ymir to fashion the world from him.  They made the first people, Ask and Embla.  Odin fashioned all things which we see, but the destructive giants opposed and fought him; the hideous blob which invaded the earth came from the giants.  As such, we are dedicated to its extermination, to drive it from the face of the Earth until we see it no more.  Only then shall we undo <the_cataclysm>, and fashion the world anew.",
```

```suggestion
    "dynamic_line": "It seems likely you wouldn't know.  Before there was anything in the world, there was fires of Muspelhiem and the frigid ice of Niflheim.  When these two came together, the ice melted and became the first of the giants, Ymir.  As he slept, more giants sprung from his sweat and saliva.  From the melt came the cow Audhumla, who licked away the ice until she uncovered Buri, the first of the gods.  His children were Odin, Vili, and Ve; who slew Ymir to fashion the world from him.  They made the first people, Ask and Embla.  Odin fashioned all things which we see, but the destructive giants opposed and fought him; these undead which invaded the earth came from Hel as an army of the dead during Ragnarok.  As foretold in Vafthrúdnismál, Vidar and some of the younger generation of gods will create a new world.  Only then shall we undo <the_cataclysm>, and fashion the world anew.",
```
FYI, the blob doesn't exist as far as any survivors are concerned, not even Xedra knows about "the blob". Therefore any content mentioning "the blob" is unmergeable.